BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -215,8 +216,25 @@ class CbrainTask::<%= name %> < <%= descriptor['cbrain:inherits-from-class'] ||
       %>
 %     end
     }
+%   end
 
+%   unless seps.empty?
+    # Input command-line separators (with flags) used with keys in command-line
+    # substitution.
+    seps = {
+%     key_width = max_width.(seps, 'command-line-key')  + "''".length
+%     seps.each do |sep|
+      <%=
+        "%-#{key_width}s => %s" % [
```

Comment:
Can any of the keys or values include a single quote character? Because if so this will break.